src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function createMessage(message, isError = false) {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.classList.add('message');
+
+  if (isError) {
+    messageDiv.classList.add('error-message');
+  }
+
+  messageDiv.textContent = message;
+  document.body.appendChild(messageDiv);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', resolve);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
```

Comment:
I think you misunderstood the task - `promise2` should be just rejected in 3 seconds and don't require a click, but that's not a big deal)